src/main/java/dk/cngroup/lentils/service/TeamService.java

```diff
@@ -31,14 +31,26 @@ public TeamService(final TeamRepository teamRepository,
         this.roleService = roleService;
     }
 
+    public Team update(final Team team) {
+        team.setPin(getUniquePin());
+        User user = team.getUser();
+        addUserDataToTeam(team, user);
```

Comment:
Jinak co se dívám na tuhle metodu a tu šablonu s th:object, asi bych to řešil jinak příště. Pro entitu a formulář mít oddělené třídy a v kontroleru bych na základě ID v URL vytáhnul ten orm model a do něho jen nasetoval položky z formuláře.

Pak se nemusí takhle "znovu nastavovat" položky na té entitě, které nejdou ve formuláři měnit.

Reply:
muze se vytvorit issue a zrefaktorovat ;)

Reply:
ja jsem ty metody jen rozsiril